imsv-docs-astro/src/content/docs/guides/authentication.mdoc

```diff
@@ -42,14 +42,14 @@ an address is based upon a process of authentication by way of verification of
 [EIP-4361](https://eips.ethereum.org/EIPS/eip-4361) challenges signed by the
 private key underlying an address. Upon successful authentication, Immersve
 will issue an access token for subsequent use in interactions with the Immersve
-API.
+API. Additionally, a refresh token will also be provided, enabling the request of a new access token when the current one expires.
 
 Subsequent requests for protected resources from the Immersve API may be
 authenticated by supplying the access token in the `Authorization` header.
 
 ### Login Flow
 
-<!-- https://miro.com/app/board/uXjVNxzJMjE=/?moveToWidget=3458764577781765320&cot=14 -->
+<!-- https://miro.com/app/board/uXjVNxzJMjE=/?moveToWidget=3458764581317527695&cot=14 -->
```

Comment:
1. Let's revert this. The refresh token logic is not part of the login sequence.
2. I'm not sure if we need to visualize the refresh token sequence at all.
3. The token exchange should initiated based on access token's exp claim not based on a failed API call.

Reply:
That was my suggestion to augment the login diagram with the refresh token flow, but I agree with @ncjones now I see it, it makes the diagram a bit too long and complicated, and I think you've explained it clearly enough in the Refresh Token section already @magsharon  👍

Reply:
Thanks to both @NicBathgate and @NicBathgate for your input. Here are the updates I've implemented:
1) I've rolled back the login sequence to its original configuration.
2) Adjustments have been made to the texts.
3) A [new refresh token sequence](https://miro.com/app/board/uXjVNxzJMjE=/?moveToWidget=3458764581391822418&cot=14) has been created in Mirror. I really believe it's important to have a diagram for it. 
4) Lastly, I've introduced a refresh token flow at the end (with the [new refresh token sequence](https://miro.com/app/board/uXjVNxzJMjE=/?moveToWidget=3458764581391822418&cot=14))